testutils/helper/install.go

```diff
@@ -101,20 +101,20 @@ func NewSoloTestHelper(configFunc TestConfigFunc) (*SoloTestHelper, error) {
 		testConfig.InstallNamespace = testConfig.version
 	}
 
+	testHelper := &SoloTestHelper{
+		TestConfig: &testConfig,
+	}
+
 	// Optionally, initialize a test runner
-	var testRunner *testRunner
 	if testConfig.DeployTestRunner {
```

Comment:
what is a test runner used for, and in what scenarios would we want / not want to use it?

Reply:
same question/do you have some more context for this change?

Reply:
heh yeah tossed this up but didn't have time to add hte context, will fill in now